pkg/activator/throttler_test.go

```diff
@@ -184,7 +191,7 @@ func TestThrottlerActivatorEndpoints(t *testing.T) {
 
 			throttler := getThrottler(
 				defaultMaxConcurrency,
-				revisionLister(testNamespace, testRevision, v1beta1.RevisionContainerConcurrencyType(s.revisionConcurrency)),
+				revisionLister(testNamespace, testRevision, ptr.Int64(int64(s.revisionConcurrency))),
```

Comment:
make 
1. revisionConcurrency int64
2. make defaultConcurrency  int64 as well
this should simplify